Cargo.toml

```diff
@@ -56,3 +56,7 @@ path = "examples/single_mem_node/main.rs"
 [[example]]
 name = "five_mem_node"
 path = "examples/five_mem_node/main.rs"
+
+[replace]
```

Comment:
Please use `[patch]` in all cases as `[replace]` doesn't work with `cargo vendor` :(